web-api/domain-s3-backup.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -e
```

Comment:
@ericsorenson Could we get a description of the purpose of this script here?

Reply:
It's a one time script for this PR, documented in https://github.com/flexion/ef-cms/issues/5666#issuecomment-667339500